src/main/java/seedu/address/logic/commands/RedoCommand.java

```diff
@@ -7,6 +7,9 @@
 
 public class RedoCommand extends Command {
     public static final String COMMAND_WORD = "redo";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Reverts the person list or task list to a later "
+            + "state, which exists after an undo command was executed.";
```

Comment:
": Reverts the person list or task list to a later "
            + "state, which exists after an undo command was executed.\n" + "Example: redo"

Sorry I missed out the example for the command usage for undo and redo. I think adding them in is good and will make it more consistent with the other commands!